TL improvement (#522)
07th-mod/higurashi-console-arcs: Update/omot_001.txt

//rそう…きっとそれは、遠い昔に私が置いてきた、とても大切で、とても大好きだったもの。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そう…きっとそれは、遠い昔に私が置いてきた、とても大切で、とても大好きだったもの。",
			NULL, "Yes... I'm sure it was something my loved, something precious that I lost long ago.", Line_Normal);
	ClearMessage();

//rそして、その証拠に私は知らないうちに両手を伸ばし、声が聞こえる先に向かって胸いっぱいの大声で呼びかけていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、その証拠に私は知らないうちに両手を伸ばし、声が聞こえる先に向かって胸いっぱいの大声で呼びかけていた。",

> 			NULL, "That's why I stretched both my hands out and boldly called out to that voice without realizing.", Line_Normal);

Author: I translated 知らないうちに as "without realizing" but "before I knew it" is also okay, I think.

Literal meaning of その証拠に is "as evidence of that." ( "to receive some proof of that" is used here) 
I think it'd be better to simply translate as "so" or "that's why" in this case.

//r親なのに。母なのに。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "親なのに。母なのに。",

> 			NULL, "Even though you're my parent. My mother.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even though you're my family. My mother.", Line_Normal);
```

//rそう、母のせいだけじゃない。言い方こそきつく、思い込みが激しいだけに独りよがりな面があったにせよ、母は母なりに私のことを心配して、そして一生懸命だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そう、母のせいだけじゃない。言い方こそきつく、思い込みが激しいだけに独りよがりな面があったにせよ、母は母なりに私のことを心配して、そして一生懸命だった。",

> 			NULL, "Right, it wasn't just my mother's fault. She was being conceited with her assumptions and the way she spoke. Still, she was just being a mother, worrying about me and doing her best to care for me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Right, it wasn't just my mother's fault. Even though she was being conceited with her assumptions and the way she spoke, she was just being a mother, worrying about me and doing her best to care for me.", Line_Normal);
```

//r羽入のことも、ゆっくり長い時間をかけて説明すれば…母なら、いつかはわかってくれたはずだと…今なら、信じられる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "羽入のことも、ゆっくり長い時間をかけて説明すれば…母なら、いつかはわかってくれたはずだと…今なら、信じられる。",

> 			NULL, "If only I had taken it more slowly while trying to explain Hanyuu to her... She's my mother after all, so I'm sure she would've come to understand one day... I can think that now.", Line_Normal);

Reviewer: ```suggestion
			NULL, "If only I had taken it more slowly while trying to explain Hanyuu to her... She's my mother after all, so I'm sure she would've come to understand one day... I can believe that now.", Line_Normal);
```

//rやはりあの時…圭一たちの……『あれ』を、見てしまったからだろうか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "やはりあの時…圭一たちの……『あれ』を、見てしまったからだろうか。",

> 			NULL, "I guess it's because... I witnessed Keiichi and the others being like \"that\"... at that time.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I guess it's because... I witnessed <i>that</i> happen to Keiichi and the others last time.", Line_Normal);
```

//r一番、頼りにならなくて…でも、一番心の安らぎをくれた同朋の姿が……。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "一番、頼りにならなくて…でも、一番心の安らぎをくれた同朋の姿が……。",

> 			NULL, "My greatest ally... who wasn't reliable at all, but who maintained my peace of mind all this time...", Line_Normal);

Reviewer: ```suggestion
			NULL, "My greatest ally... who wasn't reliable at all, but who still managed to maintain my peace of mind all this time...", Line_Normal);
```
